### ISSUE TITLE
Failed to send any messages

### ISSUE THREAD

**Issue by allanmelo** [open-wa/wa-automate-nodejs#1337]
Don't know exactly if it is related to #https://github.com/open-wa/wa-automate-nodejs/issues/1336 , but my projects are not working anymore.

Behavior:
Projects initialize, authenticate, and receive messages. However, it is not sending any kind of reply.

Actual session:
<img width="1181" alt="Captura de Tela 2021-02-11 às 20 40 04" src="https://user-images.githubusercontent.com/16151423/107713232-8a454380-6ca9-11eb-9b7a-6218ea9e6c1e.png">

The actual error I'm receiving.
<img width="1128" alt="Captura de Tela 2021-02-11 às 20 42 09" src="https://user-images.githubusercontent.com/16151423/107713265-9e894080-6ca9-11eb-9ad8-ecd087325756.png">

This same error was found in different versions and different environments:

Local environment: MacOs Catalina 10.15.7
Project 1: "@ open-wa / wa-automate": "^ 3.0.16", node v14.15.3
Project 2: "@ open-wa / wa-automate": "^ 3.1.14", node v14.15.3

Production environment: Ubuntu 20.04.2 LTS
Project 1: "@ open-wa / wa-automate": "^ 3.0.16", node v12.20.2

**Comment by allanmelo** [comment]
Just for the record:

I considered that it could be something related to some lib update because in those attempts above I had reinstalled all dependencies. So I decided to rewire an instance that ran for more than one month without errors on a Raspberry Pi, which I had disabled earlier this week. I just powered it and reauthenticated it. Unfortunately, I received the same type of error.

Environment:
<img width="1190" alt="Captura de Tela 2021-02-11 às 21 43 48" src="https://user-images.githubusercontent.com/16151423/107717652-ec567680-6cb2-11eb-9125-79985d7fa30c.png">

Problem:
<img width="865" alt="Captura de Tela 2021-02-11 às 21 42 18" src="https://user-images.githubusercontent.com/16151423/107717671-f9736580-6cb2-11eb-8cb7-aa292bcb420c.png">
